tests/mock/Concrete/PlatformCoreSetup.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Mundipagg\Core\Test\Mock\Concrete;
+
+use Mundipagg\Core\Kernel\Abstractions\AbstractModuleCoreSetup;
+use Mundipagg\Core\Kernel\Factories\ConfigurationFactory;
+use Mundipagg\Core\Kernel\ValueObjects\CardBrand;
+use Mundipagg\Core\Kernel\ValueObjects\Configuration\CardConfig;
+use PDO;
+use PDOException;
+
+class PlatformCoreSetup extends AbstractModuleCoreSetup
+{
+
+    protected function setConfig()
+    {
+        self::$config = [
+            AbstractModuleCoreSetup::CONCRETE_DATABASE_DECORATOR_CLASS =>
+                PlatformDatabaseDecorator::class,
+        ];
+    }
+
+    public function loadModuleConfigurationFromPlatform()
+    {
+        $configData = $this->getConfigMock();
+
+        $configurationFactory = new ConfigurationFactory();
+        $config = $configurationFactory->createFromJsonData($configData);
+
+        self::$moduleConfig = $config;
+    }
+
+    protected function setModuleVersion()
+    {
+        return "1.0.0";
+    }
+
+    protected function setPlatformVersion()
+    {
+        return "1.0.0";
+    }
+
+    protected function setLogPath()
+    {
+        return "/tmp/logs";
+    }
+
+    public static function getDatabaseAccessObject()
+    {
+        $memory_db = new PDO('sqlite:./tests/mock/db_test.sqlite');
```

Comment:
Apesar que não tem nada a ver com o que foi feito adicionar ext-pdo no composer se for possível

Reply:
Feito